htsget-http-lambda/src/lib.rs

```diff
@@ -493,7 +485,7 @@ mod tests {
         assert!(router.get_route(&Method::DELETE, &uri).is_none());
       },
       &config,
-      formatter_from_config(&config),
+      formatter_from_config(&config).unwrap(),
```

Comment:
I'd try to avoid this series of `.unwrap()` here and below if possible

Reply:
Yes, this one could definitely be avoided.

Although, this made me think of a different point. All our tests use unwraps to panic, however there is also support to return a [`Result`](https://doc.rust-lang.org/rust-by-example/testing/unit_testing.html#tests-and-) from a test which could lead to more informative errors and make them more concise. Do you think it would be good for our tests to return a `Result` (obviously this is for a different PR, and not super high priority)?

Reply:
Yeah, I tend to prefer avoiding `unwrap()` even in small examples for that kind of reason.